### PR TITLE
[Validator] Update Serbian translation file

### PR DESCRIPTION
**Pull request by burnmaniac**
| Q             | A
| ------------- | ---
| Branch?       | 3.4 
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | #30189  
| License       | MIT
| Doc PR        | / 

I am not sure about some translations, if someone could check these, I would appreciate it.

Also, I've found that singular/plural translations have 3 translations. I am not sure if that's a mistake or not. I removed the third translation, but if I was wrong, I'll be happy to put it back.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 1]
```diff
@@ -2,302 +2,338 @@
...
-            </trans-unit>
-            <trans-unit id="76">
-                <source>The image is landscape oriented ({{ width }}x{{ height }}px). Landscape oriented images are not allowed.</source>
-                <target>Slika je orijentacije pejzaža ({{ width }}x{{ height }}px). Pejzažna orijentacija slika nije dozvoljena.</target>
-            </trans-unit>
-            <trans-unit id="77">
-                <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
-                <target>Slika je orijantacije portreta ({{ width }}x{{ height }}px). Portretna orijentacija slika nije dozvoljena.</target>
-            </trans-unit>
+        <trans-unit id="1">
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: could you please keep the indentation as it was before?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 2]
```diff
@@ -4,39 +4,39 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Vrednost treba da bude netačna.</target>
+                <target>Vrednost bi trebalo da bude netačna.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: I think it was good before, I guess both are valid?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 3]
```diff
@@ -4,39 +4,39 @@
...
             </trans-unit>
             <trans-unit id="4">
                 <source>This value should be blank.</source>
-                <target>Vrednost treba da bude prazna.</target>
+                <target>Vrednost bi trebalo da bude prazna.</target>
             </trans-unit>
             <trans-unit id="5">
                 <source>The value you selected is not a valid choice.</source>
-                <target>Vrednost treba da bude jedna od ponuđenih.</target>
+                <target>Selektovana vrednost nije validan izbor.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Izabrana

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 4]
```diff
@@ -4,39 +4,39 @@
...
             </trans-unit>
             <trans-unit id="6">
                 <source>You must select at least {{ limit }} choice.|You must select at least {{ limit }} choices.</source>
-                <target>Izaberite bar {{ limit }} mogućnost.|Izaberite bar {{ limit }} mogućnosti.|Izaberite bar {{ limit }} mogućnosti.</target>
+                <target>Morate odabrati bar {{ limit }} opciju.|Morate odabrati bar {{ limit }} opcija.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: `Mogucnost` sounds more appropriate here.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 5]
```diff
@@ -68,39 +68,39 @@
             </trans-unit>
             <trans-unit id="17">
                 <source>The mime type of the file is invalid ({{ type }}). Allowed mime types are {{ types }}.</source>
-                <target>Mime tip datoteke nije validan ({{ type }}). Dozvoljeni mime tipovi su {{ types }}.</target>
+                <target>MIME tip datoteke nije validan ({{ type }}). Dozvoljeni MIME tipovi su {{ types }}.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: not sure if this should be caps

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 6]
```diff
@@ -68,39 +68,39 @@
...
             </trans-unit>
             <trans-unit id="22">
                 <source>This value should not be blank.</source>
-                <target>Vrednost ne treba da bude prazna.</target>
+                <target>Vrednost ne bi trebalo da bude prazna.</target>
             </trans-unit>
             <trans-unit id="23">
                 <source>This value should not be null.</source>
-                <target>Vrednost ne treba da bude null.</target>
+                <target>Vrednost ne bi trebalo da bude prazna.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Ah, hate that `null` and `blank` translates to the same word...

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 7]
```diff
@@ -160,35 +160,35 @@
...
+                <target>Vrednost bi trebalo da ima tačno {{ limit }} karakter.|Vrednost bi trebalo da ima tačno {{ limit }} karaktera.</target>
             </trans-unit>
             <trans-unit id="49">
                 <source>The file was only partially uploaded.</source>
                 <target>Datoteka je samo parcijalno otpremljena.</target>
             </trans-unit>
             <trans-unit id="50">
                 <source>No file was uploaded.</source>
-                <target>Datoteka nije otpremljena.</target>
+                <target>Nedostaje datoteka za otpremljivanje.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: I think it was valid before.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 8]
```diff
@@ -292,11 +292,47 @@
...
-                <target>Slika je orijantacije portreta ({{ width }}x{{ height }}px). Portretna orijentacija slika nije dozvoljena.</target>
+                <target>Slika je portretno orijentisana ({{ width }}x{{ height }}px). Portretno orijentisane slike nisu dozvoljene.</target>
+            </trans-unit>
+            <trans-unit id="78">
+                <source>An empty file is not allowed.</source>
+                <target>Prazna datoteka nije dozvoljena.</target>
+            </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Nije moguće odrediti poslužitelja.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: not sure if anyone can understand this translation but it is valid...

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 9]
```diff
@@ -292,11 +292,47 @@
...
+                <source>Error</source>
+                <target>Greška</target>
+            </trans-unit>
+            <trans-unit id="83">
+                <source>This is not a valid UUID.</source>
+                <target>Ovo nije validan univerzalni unikatni identifikator (UUID).</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Ova vrednost bi trebalo da bude višestruka od {{ compared_value }}.</target>
```
Comment (kunicmarko20) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: `višestruko` or `višestruka`?

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 10]
```diff
@@ -292,11 +292,47 @@
             </trans-unit>
             <trans-unit id="76">
                 <source>The image is landscape oriented ({{ width }}x{{ height }}px). Landscape oriented images are not allowed.</source>
-                <target>Slika je orijentacije pejzaža ({{ width }}x{{ height }}px). Pejzažna orijentacija slika nije dozvoljena.</target>
+                <target>Slika je pejzažno orijentisana ({{ width }}x{{ height }}px). Pejzažna orijentisane slike nisu dozvoljene.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: piksela? or if we keep "px", should we keep it above?
same question below.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf [hunk 11]
```diff
@@ -292,11 +292,47 @@
...
+                <source>The host could not be resolved.</source>
+                <target>Nije moguće odrediti poslužitelja.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Vrednost se ne poklapa sa očekivanim {{ charset }} setom karaktera.</target>
+            </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>Ovo nije validan BIC.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: don't we want to add something that'd mean "Business Identifier Code"? BIC is clear enough? (same below)

Comment (kunicmarko20) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Don't know if we have a word for BIC, maybe we can literally translate it

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf (hunk 2 above)
Comment (burnmaniac) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: The old one is understandable, but, if we want it by the rules, the new one is valid.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf (hunk 5 above)
Comment (burnmaniac) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Also not sure. I've found it in some books on Serbian it's written in full caps, which is why I changed it.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf (hunk 6 above)
Comment (burnmaniac) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Me too, but I couldn't find proper word for it.

File: src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf (hunk 9 above)
Comment (burnmaniac) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: Maybe `višestruko veća`?

Comment (kunicmarko20) [reply src/Symfony/Component/Validator/Resources/translations/validators.sr_Latn.xlf]: yes!

**Review by kunicmarko20 — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @burnmaniac.

**Pull request merged**